lib/src/widgets/chat/ds_message_bubble_detail.widget.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:blip_ds/blip_ds.dart';
+import 'package:blip_ds/src/widgets/chat/ds_delivery_report_icon.widget.dart';
+import 'package:flutter/material.dart';
+
+/// A Design System widget used to display a date and a [DSDeliveryReportIcon] at a message.
+class DSMessageBubbleDetail extends StatelessWidget {
+  final String date;
+  final DSAlign align;
+  final DSDeliveryReportStatus deliveryStatus;
+
+  /// Creates a new Design System's [DSMessageBubbleDetail]
+  const DSMessageBubbleDetail({
+    Key? key,
+    required this.date,
+    required this.align,
+    required this.deliveryStatus,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      mainAxisAlignment: align == DSAlign.right
+          ? MainAxisAlignment.end
+          : MainAxisAlignment.start,
+      children: [
+        Container(
+          padding: const EdgeInsets.fromLTRB(16.0, 4.0, 16.0, 0),
+          child: Row(
+            children: [
+              if (align == DSAlign.right) ...[
+                DSDeliveryReportIcon(deliveryStatus: deliveryStatus),
+                const SizedBox(
```

Comment:
Would it pay to create two widgets? Another one already receiving only the text and showing the set already formatted. This would simplify and avoid code repetition every time you use the delivery report.